pkg/landscaper/controllers/deployitem/reconcile_new.go

```diff
@@ -220,11 +223,12 @@ func (con *controller) isProgressingTimeoutExceeded(log logging.Logger, di *lsv1
 	return false, &requeue, nil
 }
 
-func (con *controller) writeProgressingTimeoutExceeded(ctx context.Context, log logging.Logger, di *lsv1alpha1.DeployItem) error {
+func (con *controller) writeProgressingTimeoutExceeded(ctx context.Context, di *lsv1alpha1.DeployItem) error {
 	// the deployer has not finished processing this deploy item within the timeframe
 	// => abort it
-	logger := log.WithValues("operation", "DetectProgressingTimeouts")
-	logger.Debug("deploy item timed out, setting abort operation annotation")
+	logger, ctx := logging.FromContextOrNew(ctx, nil)
+	logger = logger.WithValues(lc.KeyMethod, "writeProgressingTimeoutExceeded")
```

Comment:
```suggestion
	logger, ctx := logging.FromContextOrNew(ctx, nil, lc.KeyMethod, "writeProgressingTimeoutExceeded")
```

Reply:
Again the `KeyMethod` logging key, which is only for the current method, but should not be passed in a context to sub methods.